index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+        integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    
+    <title>FullStack ToDo</title>
+</head>
+
+<body>
+
+    <div class="user-input">
+        <input type="text" id="todo-input" placeholder="Whatdo you need to do?">
+        <button id="addToDo">Add</button>
+    </div>
+
+    <div id="todos"></div>
+
+    <script id="todos-template"  type="text/x-handlebars-template" >
+        {{#each todo}}
+        {{#if completed}}
+                <div class="todo" id="{{id}}" >
+                <span style="text-decoration: line-through;" class="text" >{{text}}</span>  
```

Comment:
Do this in a class.
And make the `if` only on this line, don't repeat yourself for everything.